include/pythontarget.h

```diff
@@ -184,6 +184,53 @@ typedef struct {
 } generic_port_capsule_struct;
 
 
+/**
+ * Special version of the template_input_output_port_struct
+ * for dynamic arrays.
+ * FIXME: This is currently kept for compatibility reasons
+ * and should be removed soon.
+ **/
+typedef struct {
+    PyObject_HEAD
+    PyObject* value;
+    bool is_present;
+    int num_destinations;
+    lf_token_t* token;
+    int length;
+} generic_port_instance_with_token_struct;
+
+/**
+ * Python wrapper for the tag_t struct in the C target.
+ **/
+typedef struct {
+    PyObject_HEAD
+    tag_t tag;
+} py_tag_t;
+
+PyTypeObject TagType;
```

Comment:
I think this line might be unnecessary. This `TagType` object is already defined in `pythontarget.c` and it is not meant to be used outside of that .c file anyway

Reply:
I put that here because some of the functions in pythontarget.c that comes before the TagType definition references the TagType. I have removed the line here and moved it to the top of pythontarget.c